.github/workflows/integration-tests.yml

```diff
@@ -11,17 +11,23 @@ jobs:
   tests:
     name: Integration tests
     runs-on: ubuntu-20.04
+    strategy:
+      max-parallel: 1
+      matrix:
+        python-version:
+        - '3.6'
+        - '3.8'
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Set up Python 3.6
+      - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
         with:
-          python-version: '3.6'
+          python-version: ${{ matrix.python-version }}
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install tox
+          python -m pip install tox==3.28.0
```

Comment:
why was this required?

Reply:
Hi Lukasz :smile:
about the `tox==3.28.0`
The latest version of tox doesn’t support python3.6, so if we not provide any version, python 3.6 will get tox 3.28.x and the python 3.8 will get tox 4.x
and it have difference in term of `passenv` config, in version 3x, we can use: `passenv = JWT_CONFIG_BASE_64 ADMIN_USER_ID` but in 4.x, we need to use `passenv = JWT_CONFIG_BASE_64,ADMIN_USER_ID`, kind of breaking change.
So in here, I specified that we only use 3.28.0 as tox version.